src/aggregator/aggregator/gauge_elem_gen.go

```diff
@@ -155,6 +157,18 @@ func (e *GaugeElem) AddUnion(timestamp time.Time, mu unaggregated.MetricUnion, r
 // AddValue adds a metric value at a given timestamp.
 func (e *GaugeElem) AddValue(timestamp time.Time, value float64, annotation []byte) error {
 	alignedStart := timestamp.Truncate(e.sp.Resolution().Window).UnixNano()
+
+	if strings.Contains(e.ID().String(), "rpc_server_request_duration_seconds_bucket") {
```

Comment:
Can we refactor this check out so that it'd be easier to monitor other metrics in the futrue?
E.g.
```
func shouldLog(e *GaugeElem) bool {...
```

Reply:
good point, let me make it to a method